robyn/testing.py

```diff
@@ -0,0 +1,35 @@
+
+# the "app" parameter in the constructor refers to a Robyn application
+# app.router leads to the router, and from there we can use app.router.get_routes()
+# to retrieve the routes the server supports. We can search said list for methods
+# to match test requests made by the server's developer
+
+# it's important to understand the following classes, at least right now:
+# Route, Request, Response
+
+import asyncio
+from robyn import Request, Response
```

Comment:
@ElyasMag try this

from robyn.robyn import Router

router = Router(...)
print router.getRoute()